dotnet-platform-specific/RetrievalOfCpuCores/Tests/RetrievalOfCpuCoresTests.cs

```diff
@@ -0,0 +1,69 @@
+namespace Tests
+{
+    [TestClass]
+    public class RetrievalOfCpuCoresTests
+    {
+        [TestMethod]
+        public void WhenGetLogicalProcessors_LogicalProcessorEqualToEnvironmentProcessorCount()
+        {
+            if (!OperatingSystem.IsWindows())
+            {
+                return;
+            }
+            var cpuInformation = new CpuInformation();
+            var logicalProcessors = cpuInformation.GetLogicalProcessors();
+            var processorsCount = cpuInformation.GetProcessorsCount();
+            Assert.AreEqual(logicalProcessors, processorsCount);
+        }
+
+        [TestMethod]
+        public void WhenGetLogicalProcessors_ResultIsGreaterThanZero()
+        {
+            if (!OperatingSystem.IsWindows())
```

Comment:
@lennart02 
would it be better if we add this logic on the `SUT` instead of in the tests

Reply:
I added an Exception in the SUT. However, this check is still necessary in the tests, as we cannot perform it on non-Windows computers.